hooks/useEasyWrite.ts

```diff
@@ -97,16 +97,12 @@ export const useEasyWrite = (params: UsePrepareContractWriteConfig) => {
     config.request,
```

Comment:
_(random placement)_
Find this a bit jarring, maybe we can keep the error while the amount is being checked and only hide the error once the latest input is correct?

https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/d39dc47b-ca86-45d5-9b8c-d0667b0d222e

Reply:
Same thing here, also not sure why there's a L shaped white space in the error box.
https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/57049642-ff46-426d-a23c-2e351dc096c5

Reply:
For the jarring error text, i added an `&nbsp;` so the space there exists even when there's no error. should make it less jarring.

can't reproduce the L shaped white space 😵